src/renderer/views/watch.js

```diff
@@ -88,25 +89,15 @@ class watch extends React.Component {
                         </div>
                         <div className="single-video-author box mb-3">
                             <div className="float-right">
-                                <button className="btn btn-light btn-sm" data-subscribe="mediamysteries" type="button">
-                                    <span id="substatus">Follow</span>
-                                    <strong className="ml-2" id="subcount">
-                                        <a href={`/user/${this.state.post_info.author}/followers`} className="view-followers" data-tooltip="tooltip" title="Click to see followers">
-                                            {
-                                                //TODO: implement folower count.
-                                            }
-                                        </a>
-                                    </strong>
-                                    <FaBell data-toggle-notifications="mediamysteries"></FaBell>
-                                </button>
+                                <Follow user={this.state.post_info.author} />
```

Comment:
Both line [92](https://github.com/vaultec81/blasio-app/pull/3/files#diff-1cb8b96859ccc084c79d4912bace9803R92)  and [100](https://github.com/vaultec81/blasio-app/pull/3/files#diff-1cb8b96859ccc084c79d4912bace9803R100) need to support reflink format.